subsys/bluetooth/controller/ll_sw/ull_llcp_phy.c

```diff
@@ -887,7 +889,8 @@ static void rp_pu_send_phy_update_ind(struct ll_conn *conn, struct proc_ctx *ctx
 				      void *param)
 {
 	if (llcp_rr_ispaused(conn) || !llcp_tx_alloc_peek(conn, ctx) ||
-	(llcp_rr_get_paused_cmd(conn) == PROC_PHY_UPDATE)) {
+	(llcp_rr_get_paused_cmd(conn) == PROC_PHY_UPDATE) ||
+	    !ull_is_lll_tx_queue_empty(conn)) {
```

Comment:
why this? Means phy update will not add tx unless tx queue is empty? That seems wrong?

Reply:
Gottit